common-values.yaml

```diff
@@ -47,3 +49,5 @@ global:
     default:
       name: default
       enabled: true
+  affinity: {}
```

Comment:
(opening thread randomly here)

It seems that the CI job for `make check-common-values` is not working, possibly because we are using the wrong `yq` dependency (see [here](https://github.com/kubewarden/helm-charts/actions/runs/9843343949/job/27174331140?pr=476#step:3:6)).

It would have catched the discrepancy with common-values.yaml and each of the other values.yaml.

Reply:
Oh, I was taking a look in the CI history. That's failing for a while already. =(

I guess the ubuntu runner is running the [python command](https://github.com/kislyuk/yq). But we are using the [go lang version](https://github.com/mikefarah/yq) in the script

Reply:
I'm opening an issue for this bug. Just to avoid mixing context here.